### PR TITLE
Differentiate poll name and resource name

### PR DESCRIPTION
**Pull request by tojofo**
What it says on the tin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,15 +45,16 @@ var ExampleRoute = Ember.Route.extend({
...
+      pollName: 'myContactsPoll',
+      resourceName: 'contacts', // a resource name
+      url: `http://some_domain.com/contacts/${contactId}` // url to fetch resource
+    });
   },
   actions: {
     willTransition: function (transition) {
       this._super(transition);
-      this.get('poll').removePoll('contacts'); // remove the resource from polling
+      this.get('poll').removePoll('myContactsPoll'); // remove the resource from polling
```
Comment (dnahodil) [inline README.md]: Comment should maybe be changed to match that we're removing poll by name, not resource

File: README.md [hunk 2]
```diff
@@ -45,15 +45,16 @@ var ExampleRoute = Ember.Route.extend({
   poll: Ember.inject.service(),
   ...
   afterModel: function (model, transition) {
-    this.get('poll').setup(
-      'contacts', // a resource name
-      `http://some_domain.com/contacts/${contact_id}` // url to fetch resource
-    );
+    this.get('poll').setup({
+      pollName: 'myContactsPoll',
```
Comment (dnahodil) [inline README.md]: Worth adding a comment saying name should be unique?

File: README.md [hunk 3]
```diff
@@ -71,15 +72,16 @@ var SomeComponent = Ember.component.extend({
       some_param: 'some_value',
       other_param: 'other_value'
     };
-    this.get('poll').setup(
-      'users', // resource name
-      `http://some_domain.com/users`, // url to fetch resource
-      query_params // query params
-    );
+    this.get('poll').setup({
+      pollName: 'usersPoll',
```
Comment (dnahodil) [inline README.md]: As above

File: README.md [hunk 4]
```diff
@@ -71,15 +72,16 @@ var SomeComponent = Ember.component.extend({
...
+      resourceName: 'users', // resource name
+      url: `http://some_domain.com/users`, // url to fetch resource
+      params: query_params // query params
+    });
   },
   willDestroy: function () {
     this._super();
-    this.get('poll').removePoll('users'); // remove resource from polling
+    this.get('poll').removePoll('usersPoll'); // remove resource from polling
```
Comment (dnahodil) [inline README.md]: As above

File: README.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ packaged as an [Ember CLI](https://github.com/stefanpenner/ember-cli) Addon.
 To install simply run
 
 ```
-ember install ember-cli-poll
+ember install https://github.com/Biteable/ember-cli-poll.git
```
Comment (dnahodil) [inline README.md]: :+1:

**Comment by dnahodil** [comment]
LGTM :shipit:

**Comment by jkburges** [comment]
Are we intending to maintain our own fork or are we looking to get this merged upstream at some point?

**Comment by tojofo** [comment]
It should get in upstream

**Pull request merged**